pkgs/data/fonts/glasstty-ttf/default.nix

```diff
@@ -0,0 +1,19 @@
+{ runCommand, fetchurl, lib }:
+runCommand "glasstty-ttf" {
+  src = fetchurl {
+    meta = with lib; {
```

Comment:
The meta should be attached to the runCommand, not the src.  Try `glasstty-tty.meta` in `nix repl` to check it's right.

Or, even better, you could just use `fetchurl`'s `postFetch` option to move the file into place.  Remember to update your hash if you do.

Reply:
Derp. I was writing this with him and I missed that too. As for the postFetch, it's harder to guarantee the sha256 as it's possibly to accidentally mutate the data, I don't like that possibility. The ttf can be compiled from source too, in the future, should we want to. I like the runCommand pattern better personally.

Reply:
What do you mean by "possibly to accidentally mutate the data"? You mean if somebody changes the postFetch without updating the hash?  Why would they do that when it's so simple?  Do you think it would get through review?

Reply:
I just mean that runCommand allows you to

1. Build stuff
2. Reference nix store paths

Whereas fetchurl + postfetch limits you to data, which can be easily modified by accident in postFetch, instead of being an immutable source. This font is practically just data, but I don't see why it should be artificially limited. It's just a matter of preference IMO.

Reply:
> as it's possible to accidentally mutate the data

This can only happen if you change the postFetch and since we are just copying files from A to B this can't "accidentally" happen and the extra runCommand derivation is not justified like always when only files Are copied from src to $out.